src/App.js

```diff
@@ -1,48 +1,42 @@
 import logo from "./logo.svg";
+import movieComponentImage from "./components/images/topgun.jpg";
 import "./App.css";
+import Movie from "./components/Movie";
+import Footer from "./components/Footer";
 
 function App() {
+  const movie = {
+    img: movieComponentImage,
+    name: "Топ Ган: Мэверик",
+    originalName: "Top Gun: Maverick",
+    rateImdb: 8.3,
+    year: 2022,
+    country: "США",
+    director: "Джозеф Косински",
+    genre: "Боевики",
+    duration: "02:10:32",
+    storyLine:
+      "Пит Митчелл - один из лучших пилотов, которому дали прозвище Мэверик. Он уже тридцать лет занимается любимым делом, но так и остается инструктором. Герой и сам не заинтересован в повышении по службе, поскольку это помешает ему проводить много времени за штурвалом самолета. В очередной группе новобранцев оказывается лейтенант Брэдли Брэдшоу, сын Ника Брэдшоу. С этого момента Питу предстоит сильно постараться, чтобы справиться с давними переживаниями. Он просто не имеет права отвлекаться от работы, поскольку ему необходимо отобрать группу лучших пилотов для выполнения особо важного задания...",
+  };
+
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
         <a
           className="App-link"
           href="https://reactjs.org"
           target="_blank"
           rel="noopener noreferrer"
         >
-          Хлопці! Вітаю вас на першому проекті!
-        </a>
-        <br></br>
-        <a
-          className="App-link secondary"
-          href="https://stackoverflow.com/questions/39478482/how-to-create-development-branch-from-master-on-github"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Це вєтка DEV
-        </a>
-        <a
-          className="App-link three"
-          href="https://www.instagram.com/zelenskiy_official/"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Green Link
-        </a>
-        <a
-          className="App-link four"
-          href="https://memesmix.net/meme/cn0k3v"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Another one 🔥🔥🔥
+          <img src={logo} className="App-logo" alt="logo" />
         </a>
       </header>
+
+      <main className="App-main">
+        <Movie {...movie} />
```

Comment:
Pass here just "movie" and in component call variable as movie.name  etc. Don't make const "Movie" again in component. 
Also, don't call const(variables) from capital letter.